react/Icon/index.jsx

```diff
@@ -35,8 +37,9 @@ const Icon = function ({ icon, width, height, style, color, className, preserveC
   </svg>
 }
 
+
 Icon.propTypes = {
-  icon: React.PropTypes.string.isRequired
+  icon: Types.oneOfType([Types.string, Types.object]).isRequired
 }
```

Comment:
Add other props even if they are not required.
See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md

Reply:
Thanks i added the properties. I presume you use this eslint configuration in your editor ? The lint command of cozy-ui only uses standard js, maybe you can add eslint ?